src/test/scala/mesosphere/marathon/raml/AppConversionTest.scala

```diff
@@ -94,6 +94,15 @@ class AppConversionTest extends UnitTest with ValidationTestLike {
     behave like convertToRamlAndBack(hostApp)
     behave like convertToProtobufThenToRAML(hostApp)
 
+    "correctly handle empty port definitions" in {
+      Given("An app definition")
```

Comment:
With such a small test, I'm inclined to believe that these given, then, when statements add very little value. Let's remove them?

Reply:
Also, in the links JIRA, it seems that port definitions wasn't specified at all. I think that the JIRA could be potentially clarified.

What we should really be testing is AppUpdate, right?